51-android.rules

```diff
@@ -440,21 +441,33 @@ ATTR{idProduct}=="70a9", GOTO="adbfast"
 #   Razr XT912
 ATTR{idProduct}=="4362", GOTO="adbfast"
 #   Moto XT1052
-ATTR{idProduct}=="2e83", GOTO="adbfast"
+#ATTR{idProduct}=="2e83", GOTO="adbfast"
 #   Moto E/G
-ATTR{idProduct}=="2e76", GOTO="adbfast"
+#ATTR{idProduct}=="2e76", GOTO="adbfast"
 #   Moto E/G (Dual SIM)
-ATTR{idProduct}=="2e80", GOTO="adbfast"
+#ATTR{idProduct}=="2e80", GOTO="adbfast"
 #   Moto E/G (Global GSM)
-ATTR{idProduct}=="2e82", GOTO="adbfast"
+#ATTR{idProduct}=="2e82", GOTO="adbfast"
 #   Moto x4
-ATTR{idProduct}=="2e81", GOTO="adbfast"
+#ATTR{idProduct}=="2e81", GOTO="adbfast"
```

Comment:
guess we're going to break a couple of eggs along the way, but I like where you're updates here are going....you got the various (adb and non-adb) documented, while much of the past patches less likely.